### PR TITLE
fix the legacy card style issue

### PR DESCRIPTION
**Pull request by alanrenmsft**
the card.css was shared by the new radioCardGroup component and the deprecated card component. but recently Aasim and Amir have made changes to it for the new style of the radio card group component. the ML extension is still using the card component.

the fix is to copy the content of previous version of card.css and create a new css file that is only used by card component.

after September release, Udeesha will replace the radio component with radio card group component in ML extension and I will remove the card component.

before:
image

after:
image

image

https://github.com/microsoft/azuredatastudio/issues/12420

### CODE REVIEW

File: src/sql/workbench/browser/modelComponents/card.component.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
-import 'vs/css!./media/card';
+import 'vs/css!./media/legacycard';
```
Comment (udeeshagautam) [inline src/sql/workbench/browser/modelComponents/card.component.ts:5]: we are sure this wont brake any new card right?

Comment (alanrenmsft) [reply src/sql/workbench/browser/modelComponents/card.component.ts:5]: yes, I am only updating the card component, the radio card group remains the same

File: src/sql/workbench/browser/modelComponents/media/legacycard.css [hunk 2]
```diff
@@ -0,0 +1,219 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.model-card {
```
Comment (Charles-Gagnon) [inline src/sql/workbench/browser/modelComponents/media/legacycard.css:6]: Hrm, but this is sharing the class name with the radio card group cards. Are you sure this isn't going to break those too?

https://github.com/microsoft/azuredatastudio/blob/a823e443934589491d95b975c4771f82ceca3ac2/src/sql/workbench/browser/modelComponents/radioCardGroup.component.html#L3

It should be safe to update the legacy cards to use a different class name for the time being so they only pick up the old styles.

**Review by Charles-Gagnon — CHANGES_REQUESTED** [review]

File: src/sql/workbench/browser/modelComponents/media/legacycard.css (hunk 2 above)
Comment (alanrenmsft) [reply src/sql/workbench/browser/modelComponents/media/legacycard.css:6]: with this change:
image

without this change:
image

Comment (alanrenmsft) [reply src/sql/workbench/browser/modelComponents/media/legacycard.css:6]: but to be safe, let me update the class names

Comment (alanrenmsft) [reply src/sql/workbench/browser/modelComponents/media/legacycard.css]: just to be absolutely safe, I replaced the class names in the card component and added -legacy suffix. tested and worked as expected

**Review by Charles-Gagnon — APPROVED** [review]

**Review by udeeshagautam — APPROVED** [review]

**Pull request merged**